scalyr_agent/agent_main.py

```diff
@@ -192,10 +193,6 @@ def __init__(self, platform_controller):
         self.__copying_manager = None
         # The current monitors manager.
         self.__monitors_manager = None
-        # The current ScalyrClientSession to use for sending requests.
```

Comment:
Agent main does not have client sessions anymore. All sessions now are created in the workers.